home.html

```diff
@@ -0,0 +1,217 @@
+            sharing and opening
+          </p>
+        </div>
+        <div class="program__detailes">
+          <img src="./images/program_icon_03.png" alt="Forum icon" />
+          <h3 class="program__text">Forum</h3>
+          <p class="main__program__description">
+            Listen to the speaker from various countries about the messages of
+            sharing and opening
+          </p>
+        </div>
+        <div class="program__detailes">
+          <img src="./images/program_icon_04.png" alt="Workshop Icon" />
+          <h3 class="program__text">Workshop</h3>
+          <p class="main__program__description">
+            Listen to the speaker from various countries about the messages of
+            sharing and opening
+          </p>
+        </div>
+        <div class="program__detailes">
+          <img src="./images/program_icon_05.png" alt="CC ignite icon" />
+          <h3 class="program__text">CC Ignite</h3>
+          <p class="main__program__description">
+            Listen to the speaker from various countries about the messages of
+            sharing and opening
+          </p>
+        </div>
+      </div> 
+           <a href="#" target="_blank" class="button"
+          >Join CC Global Summit 2015</a
+        >
+           <a href="#" target="_blank" class="see__program"
+          >SEE THE WHOLE PROGRAMM</a
+        >
+       </section>
+    <!---------------------------- End Here -------------------->
+
+    <!---------------------------Speakers Section Here -------------------->
+    <div class="featured__speaker">
+      <h2 class="featured__spk">Featured Speaker</h2>
+      <hr class="short__hr" />
+    </div>
+    <!---------------------------Dynamically Added Speakers Section Here ----------->
+    <section class="speaker"></section>
+    <!---------------------------- End Here ------------------------------->
+
+    <!---------------------------- Sponsor Section Here-------------------->
+    <section id="sponsor" class="sponsors">
+      <h3 class="partner">Partner</h3>
+      <hr class="short__hr" />
+      <ul class="sponsor__icons">
+        <li>
+          <img class="sponsor__img" src="./images/sponsor1.jpg" alt="COP 27 Egypt sponsor logo" />
+        </li>
+        <li>
+          <img class="sponsor__img" src="./images/sponsor3.jpg" alt="Coca Cola sponsor logo" />
+        </li>
+        <li>
+          <img class="sponsor__img" src="./images/sponsor4.jpg" alt="Africa Businees Forum sponsor logo" />
+        </li>
+        <li>
+          <img class="sponsor__img" src="./images/solar-sisters.png" alt="Solar Sister sponsor logo" />
+        </li>
+      </ul>
+    </section>
+    <!---------------------------- End Here ------------------------------->
+
+    <!---------------------------- Footer Section Here-------------------->
+    <footer class="footer">
+      <img class="footer__img" src="./images/logo.png" alt="United Nations Logo" />
+      <p class="footer__description">
```

Comment:
- As per Cindy's design, kindly make sure this section doesn't show up on the mobile version :recycle: 
![Screenshot from 2023-03-16 09-01-54](https://user-images.githubusercontent.com/97834160/225539947-7dd2cbcf-26e0-4aad-a870-d23c0e72f85f.png)

Reply:
Most certainly